src/audioPlayer.js

```diff
@@ -100,14 +109,49 @@ async function setPlayer() {
 }
 
 function resetControllers() {
-    AUDIO_TOGGLES.forEach((audioToggle) => {
-        const playIconPlayer = audioToggle.querySelector(
-            '[data-element=play]'
-        )
-        const pauseIconPlayer = audioToggle.querySelector(
-            '[data-element=pause]'
-        )
-        playIconPlayer.setAttribute('display', 'block')
-        pauseIconPlayer.setAttribute('display', 'none')
+    [...AUDIO_TOGGLES, ...VIDEO_TOGGLES].forEach((mediaToggle) => {
+        const playIcon = mediaToggle.querySelector('[data-element=play]')
```

Comment:
`play/pause` buttons are for the audio players and `mute/unmute` for video players?

Reply:
A lot of our original buttons use `play/pause`, but it looks like the designs are starting to shift to `mute/unmute`. I'll add a comment to deprecate the `play/pause` lines once they are not used in Webflow anymore

Reply:
Correction: audio toggles will use the `mute/unmute` icons if they autoplay (like with the interactive conversation block), but others will still use the `play/pause` icons if the player takes an action to start playing (like with the interviewer block). I'll still add a comment to clarify